Deprecate QGIS recipes

homebysix opened the pull request:
The QGIS recipes in this repository are not sufficiently distinct from other recipes in the AutoPkg org to warrant the extra maintenance they will require. This PR deprecates the QGIS recipes in this repo and points users to specific alternatives in other repos.

paul-cossey commented:
Looking at the dates of the open PR's and Issues in robperc-recipes I'm not overly convinced the repo is still maintained. 

I'll keep these active, but feel free to amend your PR in to robperc-recipes to point folks here.

homebysix commented:
Fair point — I hadn't noticed that robperc's repo hasn't seen any changes in many years. I'll dig into that.

In the meantime, I've adjusted the [robperc PR](https://github.com/autopkg/robperc-recipes/pull/12) to point people to your munki recipe, and I adjusted the parent of the munki recipe to the mlbz521-recipes download. I also added the needed input variable to the munki recipe.

Here's the verbose recipe run output after the change:

```
% autopkg run -vvq QGIS-LTR.munki.recipe 
Processing QGIS-LTR.munki.recipe...
WARNING: QGIS-LTR.munki.recipe is missing trust info and FAIL_RECIPES_WITHOUT_TRUST_INFO is not set. Proceeding...
URLDownloader
{'Input': {'url': 'https://qgis.org/downloads/macos/qgis-macos-ltr.dmg'}}
URLDownloader: No value supplied for prefetch_filename, setting default value of: False
URLDownloader: No value supplied for CHECK_FILESIZE_ONLY, setting default value of: False
URLDownloader: Storing new Last-Modified header: Sat, 21 Dec 2024 00:51:17 GMT
URLDownloader: Storing new ETag header: "6483c0d9-629bd2936d919"
URLDownloader: Downloaded ~/Library/AutoPkg/Cache/com.github.dataJAR-recipes.munki.QGIS-LTR/downloads/qgis-macos-ltr.dmg
{'Output': {'download_changed': True,
            'etag': '"6483c0d9-629bd2936d919"',
            'last_modified': 'Sat, 21 Dec 2024 00:51:17 GMT',
            'pathname': '~/Library/AutoPkg/Cache/com.github.dataJAR-recipes.munki.QGIS-LTR/downloads/qgis-macos-ltr.dmg',
            'url_downloader_summary_result': {'data': {'download_path': '~/Library/AutoPkg/Cache/com.github.dataJAR-recipes.munki.QGIS-LTR/downloads/qgis-macos-ltr.dmg'},
                                              'summary_text': 'The following '
                                                              'new items were '
                                                              'downloaded:'}}}
AppDmgVersioner
{'Input': {'dmg_path': '~/Library/AutoPkg/Cache/com.github.dataJAR-recipes.munki.QGIS-LTR/downloads/qgis-macos-ltr.dmg'}}
AppDmgVersioner: Mounted disk image ~/Library/AutoPkg/Cache/com.github.dataJAR-recipes.munki.QGIS-LTR/downloads/qgis-macos-ltr.dmg
AppDmgVersioner: BundleID: org.qgis.qgis3
AppDmgVersioner: Version: 3.34.14
{'Output': {'app_name': 'QGIS-LTR.app',
            'bundleid': 'org.qgis.qgis3',
            'version': '3.34.14'}}
EndOfCheckPhase
{'Input': {}}
{'Output': {}}
MunkiImporter
{'Input': {'MUNKI_REPO': '/Users/Shared/munki_repo',
           'pkg_path': '~/Library/AutoPkg/Cache/com.github.dataJAR-recipes.munki.QGIS-LTR/downloads/qgis-macos-ltr.dmg',
           'pkginfo': {'catalogs': ['testing'],
                       'description': 'Open Source Geographic Information '
                                      'System.',
                       'developer': 'Open Source Geospatial Foundation',
                       'display_name': 'QGIS-LTR',
                       'name': 'QGIS-LTR',
                       'unattended_install': True,
                       'unattended_uninstall': True},
           'repo_subdirectory': 'apps/QGIS-LTR'}}
MunkiImporter: No value supplied for MUNKI_REPO_PLUGIN, setting default value of: FileRepo
MunkiImporter: No value supplied for MUNKILIB_DIR, setting default value of: /usr/local/munki
MunkiImporter: No value supplied for force_munki_repo_lib, setting default value of: False
MunkiImporter: Using repo lib: AutoPkgLib
MunkiImporter:         plugin: FileRepo
MunkiImporter:           repo: /Users/Shared/munki_repo
MunkiImporter: Copied pkginfo to: /Users/Shared/munki_repo/pkgsinfo/apps/QGIS-LTR/QGIS-LTR-3.34.14.plist
MunkiImporter:            pkg to: /Users/Shared/munki_repo/pkgs/apps/QGIS-LTR/qgis-macos-ltr-3.34.14.dmg
{'Output': {'munki_importer_summary_result': {'data': {'catalogs': 'testing',
                                                       'icon_repo_path': '',
                                                       'name': 'QGIS-LTR',
                                                       'pkg_repo_path': 'apps/QGIS-LTR/qgis-macos-ltr-3.34.14.dmg',
                                                       'pkginfo_path': 'apps/QGIS-LTR/QGIS-LTR-3.34.14.plist',
                                                       'version': '3.34.14'},
                                              'report_fields': ['name',
                                                                'version',
                                                                'catalogs',
                                                                'pkginfo_path',
                                                                'pkg_repo_path',
                                                                'icon_repo_path'],
                                              'summary_text': 'The following '
                                                              'new items were '
                                                              'imported into '
                                                              'Munki:'},
            'munki_info': {'_metadata': {'created_by': 'testuser',
                                         'creation_date': datetime.datetime(2024, 12, 31, 22, 31, 15),
                                         'munki_version': '6.6.3.4704',
                                         'os_version': '15.2'},
                           'autoremove': False,
                           'catalogs': ['testing'],
                           'description': 'Open Source Geographic Information '
                                          'System.',
                           'developer': 'Open Source Geospatial Foundation',
                           'display_name': 'QGIS-LTR',
                           'installer_item_hash': '2bbfcb8bc3ea0f8cbcaa3ec539d6e886ff719a7126328c4fffa4773ee64b472a',
                           'installer_item_location': 'apps/QGIS-LTR/qgis-macos-ltr-3.34.14.dmg',
                           'installer_item_size': 1646832,
                           'installer_type': 'copy_from_dmg',
                           'installs': [{'CFBundleIdentifier': 'org.qgis.qgis3',
                                         'CFBundleName': 'QGIS-LTR',
                                         'CFBundleShortVersionString': '3.34.14',
                                         'CFBundleVersion': '3.34.14 ()',
                                         'minosversion': '10.13.0',
                                         'path': '/Applications/QGIS-LTR.app',
                                         'type': 'application',
                                         'version_comparison_key': 'CFBundleShortVersionString'}],
                           'items_to_copy': [{'destination_path': '/Applications',
                                              'source_item': 'QGIS-LTR.app'}],
                           'minimum_os_version': '10.13.0',
                           'name': 'QGIS-LTR',
                           'unattended_install': True,
                           'unattended_uninstall': True,
                           'uninstall_method': 'remove_copied_items',
                           'uninstallable': True,
                           'version': '3.34.14'},
            'munki_repo_changed': True,
            'pkg_repo_path': '/Users/Shared/munki_repo/pkgs/apps/QGIS-LTR/qgis-macos-ltr-3.34.14.dmg',
            'pkginfo_repo_path': '/Users/Shared/munki_repo/pkgsinfo/apps/QGIS-LTR/QGIS-LTR-3.34.14.plist'}}
Receipt written to ~/Library/AutoPkg/Cache/com.github.dataJAR-recipes.munki.QGIS-LTR/receipts/QGIS-LTR.munki-receipt-20241231-143116.plist

The following new items were downloaded:
    Download Path                                                                                               
    -------------                                                                                               
    ~/Library/AutoPkg/Cache/com.github.dataJAR-recipes.munki.QGIS-LTR/downloads/qgis-macos-ltr.dmg  

The following new items were imported into Munki:
    Name      Version  Catalogs  Pkginfo Path                          Pkg Repo Path                             Icon Repo Path  
    ----      -------  --------  ------------                          -------------                             --------------  
    QGIS-LTR  3.34.14  testing   apps/QGIS-LTR/QGIS-LTR-3.34.14.plist  apps/QGIS-LTR/qgis-macos-ltr-3.34.14.dmg
```
